src/hooks/useFetchPopkeapi.ts

```diff
@@ -1,6 +1,9 @@
 import { useEffect, useState } from "react"
 
 
+
+const localCache: { [key: string]: any } = {};
```

Comment:
Avoid the use of _any_

Reply:
You probably don't need this localCache, that info is already saved in the state